staging/nvidia/values.yaml

```diff
@@ -1,4 +1,23 @@
-global:
-  serviceLabels: {}
-serving:
-  domain: example.com
+grafana:
+  enabled: true
+
+nvidia-dcgm-exporter:
+  enabled: true
+  resources: {}
+  nodeSelector: {}
+  tolerations: []
+  initContainers: []
```

Comment:
could you educate me what is the relation between these fields and the corresponding ones in subchart's values.yaml?

Reply:
You will notice that each stanza corresponds to each subchart. This is how you override values for those subcharts, and I have chose to make this explicitly shown. The enabled fields along with the requirement file allow you then to disable subcharts.